src/main/java/com/corndel/cashregister/Level2.java

```diff
@@ -15,14 +15,32 @@ public class Level2 {
    */
   public static int countCoins(List<Item> drawer) {
     // TODO
-    return 0;
+    int total = 0;
+
+
```

Comment:
I do not think that these should pass the tests, you should check if they are coins before totalling. 

```java
    int total= 0;
    for (Item item : drawer) {
        if ((item.name.equals("penny")) || (item.name.equals("nickel")) || (item.name.equals("dime")) || (item.name.equals("quarter"))){
            total+= item.quantity;
        }
```